### DISCUSSION TITLE
Is possible create an delay when pressing an button ?

### DISCUSSION THREAD

**Discussion by Augusto7743** [AntiMicroX/antimicrox#964] [General]
Thanks for reading my topic.
I use an PS4 DS4 controller and AntimicroX 3.4 in Ubuntu 20.04.6.
I want create an assignment selecting PS4 DS4 triangle button to be an combo for keys "R" and "F" as being each press is keys "R" and "F" at same time.
However I want an delay 0.2 ms for pressing that assignment "RF".
For example I press triangle and happen an delay 0.2 ms and only after will assign to keys "R" and "F".
AntimicroX can to do it ?

Have an nice week.

**Comment by pktiuk** [comment]
It should be doable using advanced mapping.

**Comment by Augusto7743** [comment]
I not see an option "advanced mapping" ...
Where has that option "advanced mapping" for creating an delay for button ?
